src/Opdex.Platform.Infrastructure.Abstractions/Data/Models/MiningGovernances/MiningGovernanceNominationEntity.cs

```diff
@@ -1,11 +1,11 @@
 using Opdex.Platform.Common.Models.UInt;
 
-namespace Opdex.Platform.Infrastructure.Abstractions.Data.Models.Governances
+namespace Opdex.Platform.Infrastructure.Abstractions.Data.Models.MiningGovernances
 {
     public class MiningGovernanceNominationEntity : AuditEntity
     {
         public ulong Id { get; set; }
-        public ulong GovernanceId { get; set; }
+        public ulong MiningGovernanceId { get; set; }
```

Comment:
ahhh yikes, these will need sql changes on any Entity properties.

Reply:
Updated the table names, should be ready to go once db scripts are approved